src/Microsoft.DocAsCode.Build.RestApi/ApplyOverwriteDocumentForRestApiChild.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.RestApi
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Composition;
+    using System.Linq;
+
+    using Microsoft.DocAsCode.Build.Common;
+    using Microsoft.DocAsCode.Build.RestApi.ViewModels;
+    using Microsoft.DocAsCode.DataContracts.Common;
+    using Microsoft.DocAsCode.Plugins;
+
+    [Export(nameof(RestApiDocumentProcessor), typeof(IDocumentBuildStep))]
+    public class ApplyOverwriteDocumentForRestApiRootChild : ApplyOverwriteDocument
```

Comment:
I got your point, should merge into one class and judge the logic inside ApplyOvewriteDocument

---
In reply to: [64860219](https://github.com/dotnet/docfx/pull/336#discussion_r64860219) [](ancestors = 64860219)